alibi/tests/test_saving.py

```diff
@@ -448,8 +448,8 @@ def test_save_cfrl(cfrl_explainer, rf_classifier, iris_data):
         assert exp0.meta == exp1.meta
 
         # cfrl is determinstic
-        assert_allclose(exp0.cf["X"].astype(np.float32), exp1.cf["X"].astype(np.float32), rtol=1e-06)
-        assert_allclose(exp0.cf["class"].astype(np.float32), exp1.cf["class"].astype(np.float32), rtol=1e-06)
+        assert_allclose(exp0.cf["X"].astype(np.float32), exp1.cf["X"].astype(np.float32), atol=1e-06, rtol=0)
```

Comment:
Just for context, what are the approx magnitudes of `exp1.cf["X"]` and `exp1.cf["class"]`? I ask in order to get an idea of how much larger `atol=1e-6` is compared to `rtol=1e-6`, given that `atol + rtol * abs(desired)` is the comparision in `allclose(actual, desired, ...)`.

Reply:
between ~1.5 and ~-1.5 ish

Reply:
OK `atol` makes a lot of sense if `desired` is going negative, as I guess `rtol` gives an exceedingly small tolerance when `desired` hovers around zero?